tsconfig.json

```diff
@@ -34,6 +34,7 @@
       "buffer": ["./lib/buffer.js"],
       "child_process": ["./lib/child_process.js"],
       "cluster": ["./lib/cluster.js"],
+      "codeintegrity": ["./lib/codeintegrity.js"],
```

Comment:
we should also update the necessary files in `typings` folder for the internal APIs.